Digital_Excellence.sln

```diff
@@ -38,6 +38,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "14_JobScheduler", "JobSched
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "02_API.Tests", "API.Tests\02_API.Tests.csproj", "{7283F142-135F-4F59-AA64-2241A565B99E}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "JobScheduler.Tests", "JobScheduler.Tests\JobScheduler.Tests.csproj", "{C0523773-43F4-47E0-8EC6-6E058950462C}"
```

Comment:
add "16_" to the name, The other projects inside the solution are also numbered

Reply:
@NealGeilen We arent allowed to add numbers to the projects anymore as it will/can destroy alot regarding dockerfile project namings etc.

Reply:
16 is possible, just do not change the order of the numbers afterwards.

Reply:
@NealGeilen My bad, will add the number like you suggested